src/code/code_800E8EA0.c

```diff
@@ -1,107 +1,138 @@
 #include "global.h"
 
-void func_800E8EA0(GlobalContext* globalCtx, Actor* actor, u16 textId) {
+void Actor_ContinueText(GlobalContext* globalCtx, Actor* actor, u16 textId) {
     func_80151938(globalCtx, textId);
     actor->textId = textId;
 }
 
-s32 nop_800E8ED0(UNK_TYPE4 param_1) {
-    return 0;
+s32 Flags_GetEventChkInf(s32 flag) {
+    return false;
 }
 
-void nop_800E8EE0(UNK_TYPE4 param_1) {
+void Flags_SetEventChkInf(s32 flag) {
 }
 
-s32 nop_800E8EEC(UNK_TYPE4 param_1) {
-    return 0;
+s32 Flags_GetInfTable(s32 flag) {
+    return false;
 }
 
-void nop_800E8EFC(UNK_TYPE4 param_1) {
+void Flags_SetInfTable(s32 flag) {
 }
```

Comment:
What's the justification for naming these like this, if they're unused, and don't have nontrivial content?

Should probably note somewhere that they don't actually do anything, too.

Reply:
I'll admit nothing definitive, but my reasoning is based on `OoT`.
Just below `func_80035B18` (MM's equivalent is `func_800E8EA0`/`Actor_ContinueText`) are these four functions with the same function definitions, in the same order, and later on, after the more substantial flags functions, appears the rotation functions.